src/node.cc

```diff
@@ -3047,6 +3054,20 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--v8-options") == 0) {
       new_v8_argv[new_v8_argc] = "--help";
       new_v8_argc += 1;
+    } else if (strncmp(arg, "--cipher-list=", 14) == 0) {
```

Comment:
Paraphrasing my [previous comment](https://github.com/joyent/node/pull/14413#discussion_r27933665) for the corresponding PR targeted to v0.10.x:

I'm wondering if we _need_ to support a `--cipher-list` command line option in v0.12.x. Adding support for `--enable-legacy-cipher-list` already breaks the contract of not adding additional API in patch releases, and it seems sufficient to handle the use case of users not wanting to use the new default ciphers list.